.github/workflows/publish_docker_image.yaml

```diff
@@ -0,0 +1,85 @@
+name: Docker
+
+on:
+  push:
+    branches: [ "main" ]
+    tags: [ '*.*.*' ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
See comment on push about `main` and `master`

Reply:
Renamed this to `master`